django/db/migrations/state.py

```diff
@@ -571,7 +588,7 @@ def render(self, apps):
         # Restore managers
         body.update(self.construct_managers())
         # Then, make a Model object (apps.register_model is called in __new__)
-        return type(self.name, bases, body)
+        return metaclass(self.name, bases, body)
```

Comment:
Generated operations don't contain the `metaclass` argument, e.g.
```python
    operations = [
        migrations.CreateModel(
            name='Success',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
            ],
            bases=(test_one.models.Alpha, models.Model),
        ),
    ]
```
probably that's why it raises `TypeError` when running `migrate`:
```
  File "django/django/db/migrations/state.py", line 581, in render
    'Please specify a metaclass that is a (non-strict) subclass of the metaclasses of all bases'
TypeError: Please specify a metaclass that is a (non-strict) subclass of the metaclasses of all bases
```
``

Reply:
@felixxm Isn't this handled with 
```
            # Generate creation operation
            self.add_operation(
                app_label,
                operations.CreateModel(
                    name=model_state.name,
                    fields=[d for d in model_state.fields.items() if d[0] not in related_fields],
                    options=model_state.options,
                    bases=model_state.bases,
                    managers=model_state.managers,
                    metaclass=model_state.metaclass,
                ),
                dependencies=dependencies,
                beginning=True,
            )
```
(line 582 of `autodetector.py`)?

Reply:
You can create a sample project based on your tests and you will see a crash when running `migrate`.

Reply:
@felixxm ok I think I fixed it with the latest commit